x230/README.md

```diff
@@ -9,7 +9,7 @@ Get it from our [release page](https://github.com/merge/coreboot-x230/releases)
 * __SeaBIOS__: version [1.12.0](https://seabios.org/Releases) from 2018-11-17
 
 
-## table of contents
+## Table of contents
```

Comment:
unrelated typo fix, if at all

Reply:
True, just wanted to unify headings. Not important.